CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed links on settings About page that did not load (#1731)
 - A memory leak (#1628)
 - Fixed bug that could cause YouTube to display vertically offset from where it should be (#1719)
+- Bug that could cause YouTube to lose focus if the overlay was opened during loading (#1830)
```

Comment:
nit: -> "YouTube to be unresponsive"

Users don't know what losing focus means. :)